Makefile

```diff
@@ -74,12 +74,13 @@ test: manifests generate fmt vet envtest ## Run tests.
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager main.go
 
-NAME_PREFIX ?= settings-configuration.
+NAME_PREFIX ?= settings-
+NAME_CFG_PREFIX ?= settings-configuration.
 APIEXPORT_NAME ?= pipeline-service.io
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
-	go run ./main.go --api-export-name $(NAME_PREFIX)$(APIEXPORT_NAME) $(ARGS)
+	go run ./main.go --api-export-name $(NAME_CFG_PREFIX)$(APIEXPORT_NAME) $(ARGS)
```

Comment:
I am not sure it is the best way to address the issue. It could possibly get improved later on